test/kubens.bats

```diff
@@ -13,3 +16,90 @@ COMMAND="${BATS_TEST_DIRNAME}/../kubens"
   echo "$output">&2
   [ "$status" -eq 0 ]
 }
+
+@test "list namespaces when no kubeconfig exists" {
+  run ${COMMAND}
+  echo "$output"
+  [ "$status" -eq "1" ]
```

Comment:
I think we should use `[[` for consistency, no reason not to. 

also you used `"1"` here vs `0` above, please don't quote the numeric value.